src/app/public/modules/toggle-switch/toggle-switch.component.scss

```diff
@@ -0,0 +1,63 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+$indicator-size: 22px;
+
+.sky-toggle-switch {
+  display: inline-flex;
+  align-items: baseline;
+}
+
+.sky-toggle-switch-button {
+  border: 1px solid $sky-border-color-neutral-medium;
+  background-color: $sky-background-color-neutral-light;
+  padding: 1px;
+  border-radius: $indicator-size + 2;
+  width: $indicator-size * 2 + 2;
```

Comment:
The widths seem to be effected by a shrinking parent (or window). For example, if you load the visual test and resize the window small enough, it will squeeze the toggle switch's background. Is there any way to make it more resilient to width changes?

Reply:
Good catch. Fixed.